site/ultraviolet/assets/semantic-ui/js/ultraviolet/invenio-84.js

```diff
@@ -0,0 +1,17 @@
+import { waitForElement } from "./helpers"
+
+/**
+ * Adds a warning message to the file upload section of the deposit form
+ * https://jira.nyu.edu/browse/INVENIO-84
+ */
+export async function invenio84() {
+  const fileUploadWarning = await waitForElement(".file-upload-note div.ui.visible.warning.message")
```

Comment:
Please take a look at https://semantic-ui.com/elements/container.html#/examples. Let us use a different selector.

Example code:

```javascript
export async function invenio84() {
  const message = `Contact the NYU Libraries UltraViolet service team at <a href="mailto:uv@nyu.edu">uv@nyu.edu</a> if you have more than 100 files or 50GB.`
  const elm = await waitForElement(".file-upload-note div.column")
  if (elm) {
    const div = document.createElement("div")
    div.classList.add("ui", "visible",  "warning", "message")
    div.setAttribute("id", "invenio-84-file-limits-msg")
    div.innerHTML = `<p><i aria-hidden="true" class="warning sign icon"></i>${message}</p>`
    elm.append(div)
  }
}
```

Reply:
Cool. So, change the selector from the error message to the container of the error message and append to that instead of using `parentNode`. 👍